vaccine_feed_ingest/runners/me/maine_gov/normalize.py

```diff
@@ -74,6 +75,9 @@ def _get_contacts(site: dict) -> List[schema.Contact]:
     website_matches = re.search('href="(http.*)"', scheduling_info_raw)
     if website_matches:
         website = website_matches.group(1).split(" ")[0]
+        if website.startswith("https://houltonregional.org"):
```

Comment:
Perhaps apply this fix to all the urls instead of just the one that is currently broken?